src/components/RecurringEvent/RecurringEvent.scss

```diff
@@ -0,0 +1,59 @@
+.recurring-events-modal {
+    position: fixed;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    top: 0;
+    left: 0;
+    z-index: 998;
+    background-color: rgba(0,0,0,0.75);
+    width: 100vw;
+    height: 100vh;
```

Comment:
The height could probably be set lower, currently there is whitespace and (on Chrome) for some reason the top is so high that the close button (which is a good addition =) is partially obscured behind the address bar?